klicky.code-workspace

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Not sure what this file does? It might be an extension of vscode, but it should be safe to remove